examples/main.cpp

```diff
@@ -0,0 +1,37 @@
+#include <Arduino.h>
+
+#ifdef ESP8266
+#include <SoftwareSerial.h>
+#endif
+
+#include "sensesp_app.h"
+#include "vedirect_helper.h"
+
+#include "sensesp_app_builder.h"
+
+ReactESP app([]() {
+  SensESPAppBuilder builder;
+  sensesp_app = builder.set_standard_sensors(IP_ADDRESS) //or, IP_ADDRESS, UPTIME, NONE
```

Comment:
It seems that a lot of people have trouble connecting to the SK Server if they don't hard-code the server address and port. Also, it's much easier to get a SensESP project working completely if the wifi SSID and password are hard-coded. Would you mind adding those two things to this section, so that if this is a new user's first SensESP Project, it has a much better chance of success? If you prefer not to, I understand. But if you do add them, also please add a sentence to your README that tells them they have to modify this section for their specifics. Thanks.